html/about.html

```diff
@@ -0,0 +1,144 @@
+              <li class="mobile-visible nav-option"><a href="../index.html#join-summit">Join</a></li>
+              <li class="desktop-visible nav-option"><a href="../index.html#seewhole">Join</a></li>
+              <li class="mobile-visible nav-option"><a href="#sponsors">Sponsor</a></li>
+              <li class="desktop-visible nav-option"><a href="../index.html#sponsors">Sponsor</a></li>
+              <li class="nav-option"><a href="https://creativecommons.org/tag/news/">News</a></li>
+              <li class="nav-option"><a href="https://creativecommons.org/campaign/">CC Campaign</a></li>
+            </ul>
+          </div>
+        </nav>
+        <div id="path-icon" class="desktop-visible">
+          <a href="../index.html">
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="black" class="bi bi-house" viewBox="0 0 16 16">
+              <path fill-rule="evenodd" d="M2 13.5V7h1v6.5a.5.5 0 0 0 .5.5h9a.5.5 0 0 0 .5-.5V7h1v6.5a1.5 1.5 0 0 1-1.5 1.5h-9A1.5 1.5 0 0 1 2 13.5zm11-11V6l-2-2V2.5a.5.5 0 0 1 .5-.5h1a.5.5 0 0 1 .5.5z"/>
+              <path fill-rule="evenodd" d="M7.293 1.5a1 1 0 0 1 1.414 0l6.647 6.646a.5.5 0 0 1-.708.708L8 2.207 1.354 8.854a.5.5 0 1 1-.708-.708L7.293 1.5z"/>
+            </svg>
+          </a>
+           > <a href="#">About</a>
+        </div>
+      </div>
+      <section id="about-event" class="container-fullwidth">
+        <span class="pre-title mobile-visible">"Hello! Sharing World"</span>
+        <h1 class="title">Creative Commons Global Summit 2015</h1>
+        <hr class="hr desktop-visible">
+        <p class="introduction">
+          The Global Summit brings together the community of experts, 
+          academics, and activists who comprise the Creative Commons 
+          affiliate network in a different country every two years.
+          <br>
+          This year, we hope to expand out invitation list including 
+          organizations and individuals<span class="mobile-visible">.</span> 
+          <span class="desktop-visible">
+            who want to work with us on shared projects that advance the cause of the Commons, 
+            free culture and open knowledge.
+            <br>
+            So if you're active and engaged in the worlds of open ocntent and knowledge - free 
+            software advocates, Wikipedians, Open Knowledge, galleries, libraries, museums, 
+            archives, governments and foundations, lawyers and activits - we hope you consider 
+            joining us this year to build a stronger, more vibrant commons together.
+          </span>
+        </p>
+        <article id="contact">
+          <span>
+            Please contact us via email for any further questions
+            <br>
+            about CC Global Summit 2015!
+          </span>
+          <br>
+          <a href="#">ccsummit2015@cckorea.org</a>
+        </article>
+      </section>
+    </main>
+    <section id="about-logo" class="section">

```

Comment:
- [ ] Logo section is almost there! However, it would be better to align it to the center and the margins as the template shows:

Template:
![Captura de pantalla 2022-04-14 a la(s) 8 10 33 p  m](https://user-images.githubusercontent.com/83717285/163502150-dcc417fe-0e2a-4ff4-b3f1-198525167863.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 8 08 15 p  m](https://user-images.githubusercontent.com/83717285/163502291-7564e3e9-46b7-4b42-bd1f-e3118f7ce3fc.png)